imagecluster/postproc.py

```diff
@@ -25,11 +25,14 @@ def plot_clusters(clusters, images, max_csize=None, mem_limit=1024**3):
         have (i) enough memory, (ii) many clusters and/or (iii) large
         max(csize) and (iv) max_csize is large or None
     """
+    if len(clusters) < 1:
+        print('No cluster meets the requirements of sim and min_csize, can\'t visualize it.')
+        return
```

Comment:
This function has no kowledge of `sim` and `min_csize` used elsewhere. I'd only catch the case `clusters={}` here by throwing an exception since, while a valid output of `calc.cluster()` it makes no sense to call this function in that case.
```suggestion
    assert len(clusters) > 0, "`clusters` is empty"
```